### ISSUE TITLE
bug report

### ISSUE THREAD

**Issue by Eniac-Xie** [speedinghzl/Pytorch-Deeplab#2]
Hi, thanks for your code.

In train.py, function 'loss_calc ' compute log_softmax for parameter 'pred' , but in CrossEntropy2d, the code use F.cross_entropy which will compute log_softmax again, so I think this maybe a bug. Maybe we should delete line 103 and 105 of  train.py?

**Comment by speedinghzl** [comment]
Thanks for your remind. I went through the [cross_entropy](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L749). You are right. The cross_entropy loss combines `log_softmax` and `nll_loss`. 
And I suggest you use the CrossEntropyLoss in Pytorch 0.2.0 version instead of CrossEntropy2d in loss.py.
